lms/envs/common.py

```diff
@@ -2951,3 +2951,13 @@
 # The cache is cleared when Redirect models are saved/deleted
 REDIRECT_CACHE_TIMEOUT = None  # The length of time we cache Redirect model data
 REDIRECT_CACHE_KEY_PREFIX = 'redirects'
+
+############## Settings for Neo4j ############################
+NEO4J_CONFIG = {
+    'bolt': True,
+    'password': 'password',
```

Comment:
@jmbowman I think we could just include it in the configuration files that set `lms.auth.json` and have separate configurations for devstack or production.

@fredsmith/ @benpatterson  thoughts on that?